lib/rucio/web/rest/flaskapi/v1/exporter.py

```diff
@@ -71,6 +74,7 @@ def get(self):
 
 export_view = Export.as_view('scope')
 bp.add_url_rule('/', view_func=export_view, methods=['get', ])
+# FIXME: Add '' rule
```

Comment:
What are these for?

Reply:
Because the blueprints do not have the url prefix, adding this rule would currently break the blueprint since it would add a rule for an empty url. These are basically notes for myself I followed up on #3990